abseil submodule is built even when using system- so skip the submodule check
diff --git a/CMakeLists.txt b/CMakeLists.txt
index d5471cc..860ea0f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,7 +75,7 @@ include(cmake/init_target.cmake)
 include(cmake/generate_target.cmake)
 include(cmake/target_yasm_sources.cmake)
 include(cmake/external.cmake)
-include(cmake/libabsl.cmake)
+# include(cmake/libabsl.cmake)
 include(cmake/libcrc32c.cmake)
 include(cmake/libopenh264.cmake)
 include(cmake/libpffft.cmake)
